cpp/benchmarks/bench_shuffle_no_partition.cpp

```diff
@@ -0,0 +1,387 @@
+/**
```

Comment:
Would it make sense to do the change in `benchmark_shuffle.cpp` as an option instead?

Reply:
Sure. Maybe, I'll add it as a cmd line option.